apidoc/Titanium/Media/Media.yml

```diff
@@ -1176,6 +1176,10 @@ properties:
     type: Number
     permission: read-only
   - name: VIDEO_CONTROL_DEFAULT
+    deprecated:
+        since: '7.0.0'
+        notes: This property has been removed for iOS in Titanium SDK 7.0.0 as of the official deprecation by Apple.
+        removed: '7.0.0'
```

Comment:
Are single quotes ok in the docs? We usually use `"`. Please validate the docs locally again. cc @jawa9000

Reply:
Very few places single quotes are there. I updated to ".